example/package.json

```diff
@@ -1,23 +1,24 @@
 {
-	"name": "example",
-	"version": "0.0.1",
-	"private": true,
-	"scripts": {
-		"start": "node node_modules/react-native/local-cli/cli.js start",
-		"test": "jest"
-	},
-	"dependencies": {
-		"react": "16.0.0-alpha.12",
-		"react-native": "0.48.3",
-		"react-native-get-sms-android": "https://github.com/davidstoneham/react-native-get-sms-android.git"
-	},
-	"devDependencies": {
-		"babel-jest": "21.0.2",
-		"babel-preset-react-native": "4.0.0",
-		"jest": "21.1.0",
-		"react-test-renderer": "16.0.0-alpha.12"
-	},
-	"jest": {
-		"preset": "react-native"
-	}
+  "name": "example",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "start": "node node_modules/react-native/local-cli/cli.js start",
+    "test": "jest"
+  },
+  "dependencies": {
+    "react": "16.6.3",
+    "react-native": "0.58.6",
+    "react-native-get-sms-android": "github:mikehardy/react-native-get-sms-android"
```

Comment:
Think we can update this to point to the main repo. I've merged the PR already but we can spin up another one.

Reply:
Yes! That was an oversight on my part. By way of excuse, it is hard to fix the main module and the example at the same time without making this mistake - I noticed it was pointing to some other random repo to start, I pointed it to main in one commit then (while doing some final testing) pointed it back to me and forgot it :-)

Reply:
No worries at all.